internal/connectors/plugins/public/modulr/plugin.go

```diff
@@ -71,11 +71,30 @@ func (p Plugin) CreateBankAccount(ctx context.Context, req models.CreateBankAcco
 }
 
 func (p Plugin) CreateTransfer(ctx context.Context, req models.CreateTransferRequest) (models.CreateTransferResponse, error) {
-	return models.CreateTransferResponse{}, plugins.ErrNotImplemented
+	if p.client == nil {
+		return models.CreateTransferResponse{}, plugins.ErrNotYetInstalled
+	}
+	payment, err := p.createTransfer(ctx, req.PaymentInitiation)
+	if err != nil {
+		return models.CreateTransferResponse{}, err
+	}
+
+	return models.CreateTransferResponse{
+		Payment: *payment,
+	}, nil
 }
 
 func (p Plugin) CreatePayout(ctx context.Context, req models.CreatePayoutRequest) (models.CreatePayoutResponse, error) {
```

Comment:
I know there's another ticket for updating all the plugins, but it might make sense to convert these to `*Plugin` for consistency while we're modifying this plugin.

Reply:
Done !